index.html

```diff
@@ -3,15 +3,15 @@
 <head>
     <title>Alison's CSD120 Website</title>
         <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initail-scale=1.0">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" href="Styles/normalize.css">
-                <link rel="stylesheet" href="Styles/main.css">
-                 <link rel="stylesheet" href="Styles/mobilemain.css" media="screen and (max-width:768px)">
```

Comment:
No marks lost, but this media query isn't really necessary since you wrap your whole CSS file inside of a `@media`  query as well.